[2023-12-12T09:18:02.979+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_bipo_dag.call_api_and_process scheduled__2023-01-18T00:00:00+00:00 [queued]>
[2023-12-12T09:18:02.988+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_bipo_dag.call_api_and_process scheduled__2023-01-18T00:00:00+00:00 [queued]>
[2023-12-12T09:18:02.989+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-12-12T09:18:03.002+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): call_api_and_process> on 2023-01-18 00:00:00+00:00
[2023-12-12T09:18:03.006+0000] {standard_task_runner.py:57} INFO - Started process 3237 to run task
[2023-12-12T09:18:03.009+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'my_bipo_dag', 'call_api_and_process', 'scheduled__2023-01-18T00:00:00+00:00', '--job-id', '202', '--raw', '--subdir', 'DAGS_FOLDER/bipo_employee_info.py', '--cfg-path', '/tmp/tmp5v2vewvz']
[2023-12-12T09:18:03.011+0000] {standard_task_runner.py:85} INFO - Job 202: Subtask call_api_and_process
[2023-12-12T09:18:03.026+0000] {logging_mixin.py:154} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-12-12T09:18:03.060+0000] {task_command.py:416} INFO - Running <TaskInstance: my_bipo_dag.call_api_and_process scheduled__2023-01-18T00:00:00+00:00 [running]> on host e1a26a98da8f
[2023-12-12T09:18:03.157+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='my_bipo_dag' AIRFLOW_CTX_TASK_ID='call_api_and_process' AIRFLOW_CTX_EXECUTION_DATE='2023-01-18T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-01-18T00:00:00+00:00'
[2023-12-12T09:24:03.085+0000] {local_task_job_runner.py:294} WARNING - State of this instance has been externally set to success. Terminating instance.
[2023-12-12T09:24:03.125+0000] {process_utils.py:131} INFO - Sending 15 to group 3237. PIDs of all processes in the group: [3237]
[2023-12-12T09:24:03.128+0000] {process_utils.py:86} INFO - Sending the signal 15 to group 3237
[2023-12-12T09:24:03.150+0000] {taskinstance.py:1632} ERROR - Received SIGTERM. Terminating subprocesses.
[2023-12-12T09:24:03.303+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=3237, status='terminated', exitcode=0, started='09:18:02') (3237) terminated with exit code 0
